### PR TITLE
TDR-343 Add new Competition and Markets Authority (Cma) body and new LD 5 series

### PR DESCRIPTION
**Pull request by thanhz**

### CODE REVIEW

File: lambda/src/main/resources/db/migration/V149__create_cma_series_and_body.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
+DO $$
+    DECLARE
+        bodyUuid "Body"."BodyId"%TYPE;
+    BEGIN
+        -- Insert CMA body into the  Body Table
+        INSERT INTO "Body" ("BodyId", "Name", "Description", "TdrCode") VALUES
+            (uuid_generate_v4(), 'Competition and Markets Authority (Cma)', 'Competition and Markets Authority (Cma)', 'TDR-CMA') RETURNING "BodyId" INTO bodyUuid;
```
Comment (TomJKing) [inline lambda/src/main/resources/db/migration/V149__create_cma_series_and_body.sql:7]: Can you confirm that the abbreviation should be `Cma` and not `CMA`. This is how data will be displayed in other systems, so its important to make sure this is correct. 

Quick look at their website and they use `CMA` for the abbreviation: https://www.gov.uk/government/organisations/competition-and-markets-authority

It's something you should confirm with the transfer advisor (ie Chris)

**Review by TomJKing — COMMENTED** [review]
Looks good. Just one minor comment. Thanks

File: lambda/src/main/resources/db/migration/V149__create_cma_series_and_body.sql (hunk 1 above)
Comment (thanhz) [reply lambda/src/main/resources/db/migration/V149__create_cma_series_and_body.sql]: I've checked with Chris. The correct abbreviation is `CMA`

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**